src/status_im/commands/handlers/debug.cljs

```diff
@@ -0,0 +1,175 @@
+(ns status-im.commands.handlers.debug
```

Comment:
I think we agreed to call al "new" namespaces where events are registered `events`, not `handlers`

Reply:
I simply moved it to the most appropriate place where all other handlers for commands are. If we want to rename `status-im.commands.handlers` to `status-im.commands.events`, we can always do it separately.